src/templates/init/test/sample-integration-test.spec.ts

```diff
@@ -0,0 +1,21 @@
+describe('Sample integration tests', () => {
+  const https = require('http');
+  const server = require('../app.js');
+  const port = 3000;
+  let serverInstance:any;
```

Comment:
I would not use TypeScript for the tests in this project as it introduces a new dependency. Consider changing it to javascript.